### PR TITLE
Use GitHub actions on windows instead of appveyor

### PR DESCRIPTION
**Pull request by mattip**
github actions gives us up to 6 hours, the builds take ~1 hour each ( X3: win32, win64, win64 w/64-bit interface)

### CODE REVIEW

**Comment by mattip** [comment]
I haven't enabled and checked uploads yet

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -2,33 +2,83 @@
...
+          fi
+          echo "START_DIR=$PWD" >> $GITHUB_ENV;
+          choco install -y zip
 
-      # Runs a set of commands using the runners shell
-      - name: Run a multi-line script
+      - run: |
+          choco install -y mingw --forcex86 --force --version=7.3.0
+          refreshenv
+          echo "C:\\ProgramData\\chocolatey\\lib\\mingw\\tools\\install\\mingw32\\bin" >> $GITHUB_PATH;
```
Comment (mattip) [inline .github/workflows/build.yml:52]: This part matches the numpy/numpy build commands. The path manipulation didn't work, I should delete it.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -2,33 +2,83 @@
...
+        name: Install 32-bit mingw
+        shell: powershell
+        if: ${{ matrix.BUILD_BITS == '32' }}
+
+      - name: Build
+        run: |
+          BITS=${{ matrix.BUILD_BITS }}
+          if [ "$BITS" == "32" ]; then
+            export PATH=/c/ProgramData/chocolatey/lib/mingw/tools/install/mingw32/bin:$PATH
+          fi
```
Comment (mattip) [inline .github/workflows/build.yml:62]: This path manipulation works. The problem is that the github actions bash runner prepends `/bin/mingw64` to the path no matter what `$GITHUB_PATH` says to do.

File: .github/workflows/build.yml (hunk 1 above)
Comment (mattip) [reply .github/workflows/build.yml]: done

**Comment by mattip** [comment]
[The secret](https://github.com/MacPython/openblas-libs/pull/41/checks?check_run_id=1252508837#step:8:36) will only be available after merge, so the last part is hard to test

**Comment by mattip** [comment]
@charris, @tylerjereddy: any reason not to merge?

**Review by tylerjereddy — APPROVED** [review]
Makes sense, can't really argue against it given that our Windows/Appveyor builds are bricked by #38 and it probably won't be long before we'll need to start producing more binaries so best to know sooner than later if there are issues with this migration.

Thanks Matti

**Pull request merged**